packages/manager/apps/cloud/client/app/dbaas/logs/detail/roles/overview/logs-role-overview.html

```diff
@@ -27,36 +27,44 @@
             <div data-ng-if="ctrl.role.permissions.length > 0">
                 <div data-ng-if="ctrl.permissions.stream.length > 0">
                     <span class="pl-0 preview-list__list-heading">Stream</span>
-                    <ul
-                        data-ng-repeat="stream in ctrl.permissions.stream track by $index"
-                    >
-                        {{stream.title}}
+                    <ul class="overview pl-0">
```

Comment:
Please refer to my previous comment.
Something like this should be enough:

```suggestion
                    <ul>
```

Reply:
Fixed yes.